metadata-io/src/main/java/com/linkedin/metadata/shared/ElasticSearchIndexed.java

```diff
@@ -2,11 +2,17 @@
 
 import com.linkedin.metadata.search.elasticsearch.indexbuilder.ReindexConfig;
 
+import com.linkedin.metadata.timeseries.BatchWriteOperationsOptions;
 import java.io.IOException;
 import java.util.List;
+import javax.annotation.Nullable;
+import org.elasticsearch.index.query.QueryBuilder;
 
 
 public interface ElasticSearchIndexed {
+    String reindexAsync(String index, @Nullable QueryBuilder filterQuery, BatchWriteOperationsOptions options)
```

Comment:
Lets remove it from this interface, it is not generally a method for all indices at this time.